readme.md

```diff
@@ -113,6 +116,13 @@ Default: `false`
 
 Uppercase the first character as in `bye-bye` → `ByeBye`.
 
+##### preserveConsecutiveUppercase
+
+Type: `boolean`\
+Default: `false`
+
+Preserve consecutive uppercase characters: `foo-BAR` → `FooBAR`.
```

Comment:
I think it would be more helpful to show the transformation with `false` and `true` for the option to users can easily compare the effect it has.

Reply:
I've added an example with the value to false.